packages/react/src/components/FormattingToolbar/FormattingToolbar.tsx

```diff
@@ -20,13 +20,15 @@ import {
 import { AddTiptapCommentButton } from "./DefaultButtons/AddTiptapCommentButton.js";
 import { FileDownloadButton } from "./DefaultButtons/FileDownloadButton.js";
 import { FilePreviewButton } from "./DefaultButtons/FilePreviewButton.js";
+import { TableCellMergeButton } from "./DefaultButtons/TableCellMergeButton.js";
 import { TextAlignButton } from "./DefaultButtons/TextAlignButton.js";
 import { FormattingToolbarProps } from "./FormattingToolbarProps.js";
 
 export const getFormattingToolbarItems = (
   blockTypeSelectItems?: BlockTypeSelectItem[]
 ): JSX.Element[] => [
   <BlockTypeSelect key={"blockTypeSelect"} items={blockTypeSelectItems} />,
+  <TableCellMergeButton key={"tableCellMergeButton"} />,
```

Comment:
I would put this at the end of the formatting toolbar instead of at the start

Reply:
Hm, not sure here. I want to make the feature more known since it only ever shows for table cells. I feel like that follows the existing pattern like the file specific buttons.